src/controllers/sessionController.ts

```diff
@@ -10,13 +19,52 @@ export const getAllSession = async () => {
 };
 
 export const getSessionById = async id => {
-  return await Session.findById(id, { _id: 0, __v: 0 });
+  return await Session.find({sessionId: id}, { _id: 0, __v: 0 });
 };
 
 export const updateSessionById = async (id, body) => {
-  return await Session.findByIdAndUpdate(id, body, { new: true });
+  const session = await Session.findOne({sessionId: id});
+
+  if (!session) {
+    return null;
+  }
+
+  return await Session.findByIdAndUpdate(session._id, body, { new: true });
 };
 
 export const deleteSessionById = async id => {
-  return await Session.findByIdAndRemove(id);
+  const session = await Session.findOne({sessionId: id});
+
+  if (!session) {
+    return null;
+  }
+  
+  return await Session.findByIdAndRemove(session._id);
 };
+
+export const validateSessionData = (sessionData) => {
+  const errors: Session = {};
+
+  if (!sessionData.title) {
+    errors.title = 'Title is required';
+  }
+  if (!sessionData.timeFrom) {
+    errors.timeFrom = 'TimeFrom is required';
+  }
+  if (!sessionData.timeTo) {
+    errors.timeTo = 'TimeTo is required';
+  }
+  if (!sessionData.author) {
+    errors.author = 'Author is required';
+  }
+  if (!sessionData.group) {
+    errors.group = 'Group is required';
+  }
```

Comment:
```suggestion
  for (const [key, value] of Object.entries(sessionData)) {
    if (!value) {
      errors[key] = `${key.charAt(0).toUpperCase() + key.slice(1)} is required`;
    }
  }
```
Предлагаю такое решение

Reply:
Такое решение не выполняется